browser/src/control/Control.NavigatorPanel.ts

```diff
@@ -150,6 +150,11 @@ class NavigatorPanel extends SidebarBase {
 			'keydown',
 			function (e: KeyboardEvent) {
 				if (e.code === 'Escape') {
+					const contextMenu = document.querySelector('.context-menu-root:not([style*="display: none"])');
```

Comment:
is it possible to add event handler on context menu in jquery `on show` ? so we will prevent later emision of the event there?